src/engraving/dom/edit.cpp

```diff
@@ -4783,59 +4783,86 @@ void Score::cloneVoice(track_idx_t strack, track_idx_t dtrack, Segment* sf, cons
                 undoAddCR(toChordRest(rest), dm, dm->tick());
             }
         }
+
+        const std::vector<EngravingItem*> annotations = oseg->annotations();
+        for (EngravingItem* annotation : annotations) {
+            if (!annotation->elementAppliesToTrack(strack)) {
+                continue;
+            }
+
+            EngravingItem* newAnnotation;
+            //does a linked clone to create just this element
+            //otherwise element will be add in every linked stave
```

Comment:
I am a bit confused by this comment. Do I understand correctly that the cloneVoice function is used both for creating linked copies (as in parts) and non-linked copies (as in the explode command)? What do you mean by "does a linked clone to create just this element"?

Reply:
Yes, copied this from above.  It's wrong, I'll correct it!